tensorflow/python/ops/ctc_ops.py

```diff
@@ -47,6 +47,18 @@ def ctc_loss(inputs, labels, sequence_length,
   max(labels.indices(labels.indices[:, 1] == b, 2))
     <= sequence_length(b) for all b.
   ```
+  
+  Useful Tips:
+  
+  The ctc_loss has as input an affine projection of RNN state:
```

Comment:
The input to ctc_loss does not need to be an affine projection of RNN state.  It can be anything; so long as it's a 3-D tensor.